README.md

```diff
@@ -25,11 +26,13 @@ Do not use these apps unless you specifically know what you are doing.
 2. Add the ios targets `rustup target add aarch64-apple-ios x86_64-apple-ios`
 3. `cargo install cargo-lipo`
 4. `cargo install cbindgen`
-5. `sudo gem install cocoapods` to install cocoapods
+5. Install Ruby dev and gems (`sudo apt install -y rubygems ruby-dev`)
```

Comment:
I don't understand the implications of this, so I'll have to look it up and test.

Reply:
well, it's not really the right thing to put here ... I was just trying the thing in Debian, knowing that it wouldn't work because no Mac/Xcode ... so, this doesn't really make sense. This PR is just a draft though so it's subject to a lot of change ...

Reply:
but, ruby and gem are required which is not otherwise mentioned in the README not sure if Mac still comes with a crufty old version of ruby or not ... I was going to take it over to my HOST-side (MacOS) and try to actually build the iOS app - when I wrote the `sudo apt install -y rubygems ruby-dev` I was just faffing in my debian container

Reply:
Either macos ships with ruby/gem that is usable for this instruction or I already had them installed (also probable). But, any Mac person is going to know how to install if needed so no need to document here.